modules/ROOT/pages/faq.adoc

```diff
@@ -116,6 +116,15 @@ Ignition equivalent.
 
 https://discussion.fedoraproject.org/t/launch-faq-how-are-fedora-coreos-nodes-provisioned-can-i-re-use-existing-cloud-init-configurations/49/1[Discuss on discussion.fedoraproject.org]
 
+== What data persists across upgrades and reboots?
+
+The directories `/etc` and `/var` are mounted as read-write which lets users
+write and modify files.
+
+The directory `/etc` may be changed by deployments, but will not override user
+made changes.  The directory `/var` is never touched by upgrades or reboots.
```

Comment:
Hmm, I think "or reboots" might be misleading. `/var` can change on a reboot into a new deployment if the upgrade e.g. added new tmpfiles entries. I think the thing we want to emphasize is that it's not changed during the upgrade operation itself. Maybe something like

```suggestion
made changes.  The directory `/var` is never touched by `rpm-ostree upgrade`.
```

?

Reply:
Good point. I'd write that down as:
> The content under `/var` is left untouched by rpm-ostree when applying upgrades or rollbacks.

Reply:
Made the changes.

I'm not sure I understand :

> /var can change on a reboot into a new deployment if the upgrade e.g. added new tmpfiles entries.

Do you have an example?

Reply:
I sadly don't have a quick example at hand, but the core is: rpm-ostree won't change `/var` directly, but anything else which is part of the upgrade could.

So, if the update comes with a tmpfiles fragment or a migration script that tweaks data under `/var`, some changes may indeed happen when you upgrade/reboot and run the new logic. This is often the case for software going through schema changes for their local data.